samples/application_development/chre/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(chre_sample)
```

Comment:
Maybe write name in full, or as minimum a comment here with full name of what chre is.

Reply:
I'll be adding a full doc to this directory explaining everything